Update functions-run-local.md

joelhulen opened the pull request:
Removed the bolding around the asterisk indicating the wildcard value, within the CORS property description. When translated to the Docs site (assuming using PanDoc or similar), the \ escape character is visible, making it read like (\\\*) in its final form.

azurecla commented:
Hi __@SSalahAhmed__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

deneha commented:
@SSalahAhmed We're trying to keep Markdown format as our primary format.  This change puts the format back into HTML.  Was there a reason why it needs to be in HTML?

SSalahAhmed commented:
Sorry I wasn't aware of that. The link wasn't showing up properly, that's I made the change.

BryanLa commented:
Thanks @SSalahAhmed. Adding @kenhoff ...  Ken: would you mind making the link update on the private repo (azure-content-pr), and removing the HTML? LMK if you're OK doing that and we can close this PR.

BryanLa commented:
@kenhoff @yossibanai - any thoughts on this?

kenhoff commented:
I can take a look at this now.

kenhoff commented:
opened a pr: https://github.com/Azure/azure-content-pr/pull/15367

closing.

BryanLa commented:
Closing per above

joelhulen committed:
Update functions-run-local.md

Removed the bolding around the asterisk indicating the wildcard value, within the CORS property description. When translated to the Docs site (assuming using PanDoc or similar), the \ escape character is visible, making it read like (\*) in its final form.
